src/app/gps/qgsgpsinformationwidget.h

```diff
@@ -105,6 +116,14 @@ class QgsGPSInformationWidget: public QWidget, private Ui::QgsGPSInformationWidg
     QFile *mLogFile = nullptr;
     QTextStream mLogFileTextStream;
     QColor mTrackColor;
+    QIntValidator *acquisitionIntValidator;
```

Comment:
Member variables need a "m" prefix to differentiate from local variables

Reply:
Also pointer variables should be initialized in the header to nullptr (or better, use std::unique_ptr where appropriate)